R/init-package.R

```diff
@@ -23,8 +22,27 @@ initPackage <- function() {
 
 addPackageLibToPath <- function() {
   libPath <- system.file("lib", package = ospsuiteEnv$packageName)
+  addPathToSystemPath(libPath)
+}
+
+addPathToSystemPath <- function(pathToAdd){
+  if (file.exists(pathToAdd)) {
+    Sys.setenv(path = paste(pathToAdd, Sys.getenv("path"), sep = ";"))
+  }
+}
 
-  if (file.exists(libPath)) {
-    Sys.setenv(path = paste(libPath, Sys.getenv("path"), sep = ";"))
+#' Loads the PKSim.R that will enable create individual and create population workflows.
+#' @param pksimFolderPath Path where PK-Sim is installed. If this is not specified, path will be read from registry using the package version
+#'
+#' @note  This will only work on Windows machine and should not be called on any other OS
+#'
+#' @import rClr
+initPKSim <- function(pksimFolderPath = NULL) {

```

Comment:
Here we need to implement something to retrieve the path from regitry. This could be adapted from the code https://github.com/PavelBal/rDotNet_feasibility/blob/master/getPathToPKSimInstallDir.R

However initPKSIm cannot be called on load. This could be called in createIndividual for example (to see)

Reply:
Dominik implemented a more sofisticated solution here:
https://github.com/Open-Systems-Pharmacology/R-Toolbox/commit/3d00c2d4c1171aa69d224d2586d5cbce75aa716b